### PR TITLE
DYN-5643 ml confidence

### PR DESCRIPTION
**Pull request by jesusalvino**
### Purpose

Implement the improvement https://jira.autodesk.com/browse/DYN-5643

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers
@QilongTang 

### FYIs
@Enzo707

### CODE REVIEW

**Comment by jesusalvino** [comment]
autocomplete-tooltip-2

File: src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml [hunk 1]
```diff
@@ -3650,7 +3650,7 @@
                                VerticalAlignment="Center"
                                HorizontalContentAlignment="Center"
                                VerticalContentAlignment="Center"
-                               Content="✓"
+                               Content="âœ“"
```
Comment (QilongTang) [inline src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml:3653]: What are these changes for?

Comment (jesusalvino) [reply src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml]: local encoder

**Review by QilongTang — APPROVED** [review]

File: src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml [hunk 2]
```diff
@@ -142,7 +144,7 @@
                         </ListBox.ItemTemplate>
                     </ListBox>
                 </Border>
-
+
```
Comment (zeusongit) [inline src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml:147]: spaces

**Pull request merged**